Create 2021-02-11-pif-18f-usds.md (#5)
wslack/wslack.github.io: _posts/2021-02-11-pif-18f-usds.md

---
title: Comparing USDS, 18F, and PIF
author: Will Slack
layout: post
permalink: /pif-18f-usds/
---

> This post seeks to contrast USDS, 18F, and the Presidential Innovation Fellows (PIF) program using public information, leveraging my experience working at 18F alongside PIFs and later at USDS. I won't cover the the many wonderful shared aspects of the programs, but briefly, all three have staff appointed to term-limited positions, share a common goal of improving federal government tech capacity, and seek to elevate and empower career civil servant partners.

Reviewer: double `the` in second sentence.

Author: ```suggestion
This post seeks to contrast USDS, 18F, and the Presidential Innovation Fellows (PIF) program using public information, leveraging my experience working at 18F alongside PIFs and later at USDS. I won't cover the many wonderful shared aspects of the programs, but briefly, all three have staff appointed to term-limited positions, share a common goal of improving federal government tech capacity, and seek to elevate and empower career civil servant partners.
```

> I should also mention two organizations doing related digital work: [Technology Transformation Services (TTS) Solutions](https://www.gsa.gov/about-us/organization/federal-acquisition-service/technology-transformation-services/tts-solutions), which manages a variety of programs such as [usa.gov](https://usa.gov), [cloud.gov](https://cloud.gov), [data.gov](https://data.gov), and [login.gov](https://login.gov), is mainly composed of career civil servants, and the [TTS Centers of Excellence](https://coe.gsa.gov/), which is set up in portfolios like AI and IT Infrastructure Optimization. All of these organizations changed under both Obama and Trump, and while I have no inside information, it’s likely there will be more change in this administration. If you’re interested in being part of the future of any of these organizations, please apply! 

Author: ```suggestion
I should also mention two organizations doing related cross-gov digital work: [Technology Transformation Services (TTS) Solutions](https://www.gsa.gov/about-us/organization/federal-acquisition-service/technology-transformation-services/tts-solutions), which manages a variety of programs such as [usa.gov](https://usa.gov), [cloud.gov](https://cloud.gov), [data.gov](https://data.gov), and [login.gov](https://login.gov), is mainly composed of career civil servants, and the [TTS Centers of Excellence](https://coe.gsa.gov/), which is set up in portfolios like AI and IT Infrastructure Optimization. All of these organizations changed under both Obama and Trump, and while I have no inside information, it’s likely there will be more change in this administration. If you’re interested in being part of the future of any of these organizations, please apply! 
```

> I should also mention two organizations doing related digital work: [Technology Transformation Services (TTS) Solutions](https://www.gsa.gov/about-us/organization/federal-acquisition-service/technology-transformation-services/tts-solutions), which manages a variety of programs such as [usa.gov](https://usa.gov), [cloud.gov](https://cloud.gov), [data.gov](https://data.gov), and [login.gov](https://login.gov), is mainly composed of career civil servants, and the [TTS Centers of Excellence](https://coe.gsa.gov/), which is set up in portfolios like AI and IT Infrastructure Optimization. All of these organizations changed under both Obama and Trump, and while I have no inside information, it’s likely there will be more change in this administration. If you’re interested in being part of the future of any of these organizations, please apply! 

Reviewer: Wonder if it makes sense to call out DDS as well in this section.

PIFs are embedded within federal agencies as “entrepreneurs in residence” for 12 months, with the opportunity to extend for a total of 2 years. Unlike USDS and 18F’s regular hiring, PIF is a fellowship that recruits annual cohorts of 20-30 fellows. Some PIF projects span fellows and cohorts (i.e. VA software portfolio to deliver veterans’ benefits faster); others are fellow-specific (i.e. global entrepreneurship program to fight poverty with Millennium Challenge Corporation).

### Parent agency

USDS is based in the [Executive Office of the President](https://www.whitehouse.gov/administration/executive-office-of-the-president/), specifically in the [Office of Management and Budget (OMB)](https://www.whitehouse.gov/omb/), which oversees executive branch agencies and assembles the yearly budget request for Congress’s consideration. Being part of "The White House" can make USDS seem intimidating, though it can also open doors. USDS's office is in a [converted 1870s town house](https://www.federaltimes.com/it-networks/2018/07/25/inside-the-agency-where-you-wish-you-worked/) once briefly occupied by President Theodore Roosevelt.

> 18F and PIF are based in the General Services Administration, a support agency for the executive branch (for example, it manages most federal real estate and manages large contract vehicles). 18F and PIF have hoteled office space in GSA's headquarters a block from the White House at 1800 F street (the [origin of 18F's name](https://18f.gsa.gov/about/#history-and-funding)).

Reviewer: Capitalize `Street`

> 18F and PIF are based in the General Services Administration, a support agency for the executive branch (for example, it manages most federal real estate and manages large contract vehicles). 18F and PIF have hoteled office space in GSA's headquarters a block from the White House at 1800 F street (the [origin of 18F's name](https://18f.gsa.gov/about/#history-and-funding)).

Author: ```suggestion
18F and PIF are based in the General Services Administration, a support agency for the executive branch (for example, it manages most federal real estate and manages large contract vehicles). 18F and PIF have hoteled office space in GSA's headquarters a block from the White House at 1800 F Street (the [origin of 18F's name](https://18f.gsa.gov/about/#history-and-funding)).
```

- Thanks to their "in residence" nature, PIFs can flexibly jump to other projects within their host agencies in a more "free agent" style over the course of their terms.

### Collaboration tools and tech

USDS often works with agencies just getting started with cloud tooling such as Microsoft Teams, so a USDSer enjoys all of the same frustrations as their career civil servant partners when trying to collaborate on a document or access shared information. GSA uses GSuite across its enterprise (the GSA IT team made the agency a [first-mover to cloud email](https://www.gsa.gov/about-us/newsroom/news-releases/gsa-becomes-first-federal-agency-to-move-email-to-the-cloud-agencywide) in 2010), so 18F and PIF teams enjoy real-time collaboration on documents with anyone at GSA. Like USDS, PIF teams get equipment and e-mail addresses from the agencies where they are deployed - there can be significant friction if you try to work with one agency using a laptop from a different agency.

### Engagement incubation

> USDS’s appropriation allows it to jump into supporting an agency on a problem immediately, though work often starts with a [discovery sprint](https://www.linkedin.com/pulse/government-discovery-sprint-playbook-how-lessons-learned-kathy-pham/). 18F sets up a formal agreement before working on an engagement, which requires signoff from multiple partner officials and in my experience, usually took over a month to execute. PIFs are deployed to agencies annually, and agencies submit projects for PIFs months in advance to be considered for that year’s class of fellows under a similar agreement to 18F's. It’s also common for PIFs to flex to other special projects once they start at an agency, based on needs that come up throughout the year.

Author: ```suggestion
USDS’s appropriation allows it to jump into supporting an agency on a problem immediately, though work often starts with a [discovery sprint](https://www.linkedin.com/pulse/government-discovery-sprint-playbook-how-lessons-learned-kathy-pham/). 18F [sets up a formal agreement](https://18f.gsa.gov/how-we-work/) before working on an engagement, which requires sign-off from multiple partner officials and in my experience, usually took over a month to execute. PIFs are deployed to agencies annually, and agencies submit projects for PIFs months in advance to be considered for that year’s class of fellows under a similar agreement to 18F's. It’s also common for PIFs to flex to other special projects once they start at an agency, based on needs that come up throughout the year.
```

> USDS’s appropriation allows it to jump into supporting an agency on a problem immediately, though work often starts with a [discovery sprint](https://www.linkedin.com/pulse/government-discovery-sprint-playbook-how-lessons-learned-kathy-pham/). 18F sets up a formal agreement before working on an engagement, which requires signoff from multiple partner officials and in my experience, usually took over a month to execute. PIFs are deployed to agencies annually, and agencies submit projects for PIFs months in advance to be considered for that year’s class of fellows under a similar agreement to 18F's. It’s also common for PIFs to flex to other special projects once they start at an agency, based on needs that come up throughout the year.

Reviewer: Worth adding a link to 18F's `How we work` in this section? https://18f.gsa.gov/how-we-work/

Author: great call!
